lib/ansible/module_utils/ovirt.py

```diff
@@ -143,6 +143,7 @@ def create_connection(auth):
         insecure=auth.get('insecure', False),
         token=auth.get('token', None),
         kerberos=auth.get('kerberos', None),
+        headers=dict(filter=auth.get('filter', False)),
```

Comment:
In order to work that properly in 4.2 where you don't have specify Filter header to user behave as user, we have to don't send anything if user didn't pass that. So I would suggest to have parameter `headers`, which is dictionary, rather then `Filter`.